src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -239,6 +244,25 @@ public static TaskStatus parseStatus(String status) throws ParseException {
         return TaskStatus.of(trimmedStatus);
     }
 
+    /**
+     * Parses a {@code String response} into a boolean.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code status} is invalid.
+     */
+    public static boolean parseYesNoResponse(String response) throws ParseException {
```

Comment:
I am not too sure if parsing into a boolean is appropriate for the ParserUtil class here because I feel that the parser is more to convert the String to create a new object rather than returning true or false. Maybe you can transfer these checks into the FilterTasksCommandParser.